src/util/math.hpp

```diff
@@ -19,4 +19,19 @@ namespace util
 		return upper3x3;
 	}
 
+	// https://stackoverflow.com/a/3407254
+	//! Round the input number to the nearest multiple of the specified number
+	/*! \param input Number to round. 
+	 *  \param multiple Multiple to round the input to. */
+	inline std::uint32_t RoundUpToNearestMultiple(std::uint32_t input, std::uint32_t multiple)
```

Comment:
We have a size align function some where I believe. d3d12_defines.hpp. its a shitty location I know.

Reply:
I'll change to code to make use of that function instead. Thanks!